users/src/test/java/org/spine3/users/UserAccountsShould.java

```diff
@@ -32,6 +32,9 @@ public void return_absent_for_google_uid_if_no_linked_account_added() {
         assertFalse(UserAccounts.getGoogleUid(UserAccount.getDefaultInstance()).isPresent());
     }
 
+    @SuppressWarnings("ConstantConditions")  /* As the {@code DUNBARS_NUMBER} value may change
+                                                a bit, it's still required to stay in certain
+                                                bounds */
```

Comment:
What about splitting the comment into 2:
 1) Describing the suppression:
`Checking the bounds of a constant value.` or smth. similar.

2) Describing the constant itself (the currently existing comment would work here).